mgmtsystem/ec2.py

```diff
@@ -55,10 +61,30 @@ def __init__(self, **kwargs):
         username = kwargs.get('username')
         password = kwargs.get('password')
 
+        for elb_region in elb_regions():
```

Comment:
this is reptetive in multiple way, i propose creating a single function that takes the elb/sqs/cloudformation modules as parameter and invokes the operations on the module